docs/configuration.md

```diff
@@ -0,0 +1,267 @@
+| `minio`     |                        | `{ "accessKey": "YOUR_MINIO_ACCESS_KEY", "secretKey": "YOUR_MINIO_SECRET_KEY", "endpoint": "YOUR_MINIO_HOST", port: 9000, secure: true }` | When `imageUploadType` is set to `minio`, you need to set this key. Also check out our [Minio Image Upload Guide](guides/minio-image-upload.md) |
+|             | `CMD_MINIO_ACCESS_KEY` | no example                                                                                                                                | Minio access key                                                                                                                                |
+|             | `CMD_MINIO_SECRET_KEY` | no example                                                                                                                                | Minio secret key                                                                                                                                |
+|             | `CMD_MINIO_ENDPOINT`   | `minio.example.org`                                                                                                                       | Address of your Minio endpoint/instance                                                                                                         |
+|             | `CMD_MINIO_PORT`       | `9000`                                                                                                                                    | Port that is used for your Minio instance                                                                                                       |
+|             | `CMD_MINIO_SECURE`     | `true`                                                                                                                                    | If set to `true` HTTPS is used for Minio                                                                                                        |
+
+### Lutim
+
+| config file | environment     | example value               | description                                                                 |
+| ----------- | --------------- | --------------------------- | --------------------------------------------------------------------------- |
+| `lutim`     |                 | `{"url": "YOUR_LUTIM_URL"}` | When `imageUploadType` is set to `lutim`, you can setup the lutim url       |
+|             | `CMD_LUTIM_URL` | `https://framapic.org/`     | When `CMD_IMAGE_UPLOAD_TYPE` is set to `lutim`, you can setup the lutim url |
+
+**Note:** *Due to the rename process we renamed all `HMD_`-prefix variables to be `CMD_`-prefixed. The old ones continue to work.*
+
+**Note:** *relative paths are based on CodiMD's base directory*
```

Comment:
Maybe move this to the 'CodiMD paths stuff' section as it seems to be most relevant there? And then remove the footnote superscripts in that section.

Reply:
I moved the note about path from ` docs/configuration.md` to `docs/url-scheme.md` in b9c75a83f779a531bae666ce1b1afdb80cbee578.

I hope it was the file that you were talking about.

Reply:
No, I was actually talking about [this section](https://github.com/oupala/server/blob/master/docs/configuration.md#L36) in the same file.

Reply:
Ok, fixed in 46e86b1be3e81956dd9e42604f370f380fa1b63b.